app/Http/Controllers/AdminController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\AddAdminRequest;
+use App\Http\Requests\UpdateAdminRequest;
+use App\Http\Resources\PermissionResource;
+use App\Models\Admin;
+use App\Services\AdminService;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class AdminController extends Controller
+{
+    protected $adminService;
+
+    public function __construct(AdminService $adminService)
+    {
+        $this->adminService = $adminService;
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
```

Comment:
Please write the list admins api as it's required for the client-side so he can choose one to edit and delete .. etc